ufm-stream-to-fluentd/README.md

```diff
@@ -0,0 +1,45 @@
+UFM stream To FluentD endpoint
+--------------------------------------------------------
+
+
+This plugin is used to reflect topology changes with time provided by UFM towered FluentD endpoint.
```

Comment:
extract topology via ufm sdk api and stream it via fluentd (link) protocol to telemetry console